sig-scalability/thresholds.md

```diff
@@ -0,0 +1,96 @@
+| Number of pods per node<sup>1</sup> | 100              |             | 100            |
+| Number of pods per core<sup>1</sup> | 10               |             | 10             |
+| Number of namespaces (ns)           | 10000            |             | 100000         |
+| Number of pods per ns               | 15000            |             | 50000          |
+| Number of services                  | 10000            |             | 100000         |
+| Number of all services backends     | TBD              |             | 500000         |
+| Number of backends per service      | 5000             |             | 5000           |
+| Number of deployments per ns        | 20000            |             | 10000          |
+| Number of pods per deployment       | TBD              |             | 10000          |
+| Number of jobs per ns               | TBD              |             | 1000           |
+| Number of daemon sets per ns        | TBD              |             | 100            |
+| Number of stateful sets per ns      | TBD              |             | 100            |
+| Number of secrets per ns            | TBD              |             | TBD            |
+| Number of secrets per pod           | TBD              |             | TBD            |
+| Number of config maps per ns        | TBD              |             | TBD            |
+| Number of config maps per pod       | TBD              |             | TBD            |
+| Number of storageclasses            | TBD              |             | TBD            |
+| Number of roles and rolebindings    | TBD              |             | TBD            |
+
+There are also thresholds for other types, but for those the numbers depend
+also on the environment (bare metal or which cloud provider) the cluster is
+running in. These include:
+
+| Quantity                                  | Master threshold | 1.8 release | Long term goal |
+|-------------------------------------------|------------------|-------------|----------------|
+| Number of ingresses                       | TBD              |             | TBD            |
+| Number of PersistentVolumes               | TBD              |             | TBD            |
+| Number of PersistentVolumeClaims per ns   | TBD              |             | TBD            |
+| Number of PersistentVolumeClaims per node | TBD              |             | TBD            |
+
+
+The rationale for some of those numbers:
+1. Total number of objects <br/>
+There is a limitation on the total number of objects on the system, as this
+affects among others etcd and its resource consumption.
+1. Number of nodes <br/>
+We believe that having clusters with more than 5000 nodes is not the best
+option and users should consider splitting into multiple clusters. However,
+we may consider bumping the long term goal at some time in the future.
+1. Number of services and endpoints <br/>
```

Comment:
Are iptables used in all configurations with all networking providers? (I really don't know)